script.tag-generator/changelog.txt

```diff
@@ -0,0 +1,44 @@
+v0.7 (December 12th, 2018)
```

Comment:
Please drop from the submission if possible

Reply:
done - i'll keep it in my repo; no need to submit to the official